### PR TITLE
Release 2.4.10

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by patriknw** [comment]
LGTM

**Comment by johanandren** [comment]
LGTM

File: news/_posts/2016-09-07-akka-2.4.10-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+author: Martynas Mickevičius
+short: Akka 2.4.10 Released!
+---
+Dear hAkkers,
+
+We—the Akka committers—are pleased to be able to announce the availability of Akka 2.4.10.
+This is mostly a bugfix release with a couple of new features. Most notable changes are:
+
+* An addition of MergeHub and BroadcastHub which enables dynamic fan-in and fan-out streams.
+Find more information in the [ticket](https://github.com/akka/akka/issues/20890) and [documentation](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-dynamic.html#Dynamic_fan-in_and_fan-out_with_MergeHub_and_BroadcastHub)
```
Comment (ktoso) [inline news/_posts/2016-09-07-akka-2.4.10-released.md]: add java link

**Comment by ktoso** [comment]
LGTM

File: news/_posts/2016-09-07-akka-2.4.10-released.md (hunk 1 above)
Comment (2m) [inline news/_posts/2016-09-07-akka-2.4.10-released.md]: Thanks!

**Pull request merged**